properties/internal/utils/struct_test.go

```diff
@@ -17,132 +17,146 @@ package utils
+	. "github.com/onsi/gomega"
 )
 
 func TestStructTag(t *testing.T) {
+
 	t.Run("Skip with empty key", func(t *testing.T) {
+		g := NewGomegaWithT(t)
+
 		st := &StructTag{
 			Key:       "",
 			OmitEmpty: false,
 			Default:   "default",
 		}
 
-		if !st.Skip() {
-			t.Errorf("Skip method should return false if the key of the struct tag is an empty string!")
-		}
+		g.Expect(st.Skip()).Should(BeTrue(),
+			"Skip method should return false if the key of the struct tag is an empty string!")
 	})
 
 	t.Run("Skip with - key", func(t *testing.T) {
+		g := NewGomegaWithT(t)
+
 		st := &StructTag{
 			Key:       "-",
 			OmitEmpty: false,
 			Default:   "default",
 		}
 
-		if !st.Skip() {
-			t.Errorf("Skip method should return false if the key of the struct tag is a `-` string!")
-		}
+		g.Expect(st.Skip()).Should(BeTrue(),
+			"Skip method should return false if the key of the struct tag is a `-` string!")
 	})
 
 	t.Run("Skip with valid key", func(t *testing.T) {
+		g := NewGomegaWithT(t)
+
 		st := &StructTag{
 			Key:       "valid.key",
 			OmitEmpty: false,
 			Default:   "default",
 		}
 
-		if st.Skip() {
-			t.Errorf("Skip method should return true if the key of the struct tag is a valid string!")
-		}
+		g.Expect(st.Skip()).Should(BeFalse(),
+			"Skip method should return true if the key of the struct tag is a valid string!")
 	})
 }
 
 func TestStructTagFlag(t *testing.T) {
 	t.Run("IsValid with empty key", func(t *testing.T) {
+		g := NewGomegaWithT(t)
+
 		stf := &structTagFlag{
 			Key:   "",
 			Value: "",
 		}
 
-		if stf.IsValid() {
-			t.Errorf("IsValid method should return false if the key of the struct tag flag is an empty string!")
-		}
+		g.Expect(stf.IsValid()).Should(BeFalse(),
+			"IsValid method should return false if the key of the struct tag flag is an empty string!")
 	})
 
 	t.Run("IsValid with non-empty key", func(t *testing.T) {
+		g := NewGomegaWithT(t)
+
 		stf := &structTagFlag{
 			Key: "omitempty",
 		}
 
-		if !stf.IsValid() {
-			t.Errorf("IsValid method should return true if the key of the struct tag flag is a non-empty string!")
-		}
+		g.Expect(stf.IsValid()).Should(BeTrue(),
+			"IsValid method should return true if the key of the struct tag flag is a non-empty string!")
 	})
 }
 
 func TestParseStructTagFlag(t *testing.T) {
 	t.Run("Empty string", func(t *testing.T) {
-		if _, err := parseStructTagFlag(""); err == nil {
-			t.Errorf("Parsing an empty string as struct tag flag should yield an error!")
-		}
+		g := NewGomegaWithT(t)
+
+		_, err := parseStructTagFlag("")
+
+		g.Expect(err).Should(HaveOccurred(),
+			"Parsing an empty string as struct tag flag should yield an error!")
 	})
 
 	t.Run("Bool flag", func(t *testing.T) {
+		g := NewGomegaWithT(t)
+
 		expected := &structTagFlag{
 			Key: "omitempty",
 		}
+
 		stf, err := parseStructTagFlag("omitempty")
 
-		if err != nil {
-			t.Errorf("Parsing a non-empty struc tag flag string should not return an error!")
-		}
+		g.Expect(err).Should(Succeed(),
```

Comment:
In many places I saw this `g.Expect(err).Should(Succeed())` expression, I think the original intention with Succeed() is to use it in pair with an `Expect(FUNCTION())`. 

I would suggest using `g.Expect(err).Should(BeNil())`, so it's more expressive for go developers.